docs/modules/Task.ts.md

```diff
@@ -595,7 +595,9 @@ Equivalent to `ReadonlyArray#sequence(ApplicativePar)`.
 **Signature**
 
 ```ts
-export declare const sequenceArray: <A>(arr: readonly Task<A>[]) => Task<readonly A[]>
+export declare const sequenceArray: <A, B>(
+  arr: readonly Task<A>[] & { readonly 0?: Task<B> | undefined }
+) => Task<Task<B> extends Task<A> ? ReadonlyNonEmptyArray<A> : readonly A[]>
```

Comment:
This signature is probably confusing to read...

Reply:
Yeah, I prefer two distinct functions with clean signatures